tomcat/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -0,0 +1,138 @@
+package com.techcourse.controller;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.UUID;
+import com.techcourse.db.InMemoryUserRepository;
+import org.apache.coyote.http11.Http11Processor;
+import org.apache.coyote.http11.HttpCookie;
+import org.apache.coyote.http11.HttpRequest;
+import org.apache.coyote.http11.HttpResponse;
+import org.apache.coyote.session.Session;
+import org.apache.coyote.session.SessionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoginController extends AbstractController {
+
+    private static final String RESOURCE_BASE_PATH = "static";
+    private static final Logger log = LoggerFactory.getLogger(Http11Processor.class);
+
+    @Override
+    protected void doGet(HttpRequest request, HttpResponse.HttpResponseBuilder response) throws Exception {
+        String resource = ensureHtmlExtension(request.getPath());
+        String responseBody = loadResourceContent(resource);
+        if (request.containsCookie()) {
+            HttpCookie httpCookie = new HttpCookie(request.getCookie());
+            handleCookieRequest(httpCookie, responseBody, response);
+        }
+
+        if (!request.containsCookie()) {
+            buildOkResponse(responseBody, response);
+        }
+    }
+
+    @Override
+    protected void doPost(HttpRequest request, HttpResponse.HttpResponseBuilder response) {
+        String requestBody = request.getRequestBody();
+        String account = getParameter(requestBody, "account");
+        String password = getParameter(requestBody, "password");
+
+        if (findUserByInfo(account, password)) {
+            handleSuccessfulLogin(response, account);
+        }
+
+        if (!findUserByInfo(account, password)) {
+            handleFailedLogin(response);
+        }
+    }
+
+    private void handleCookieRequest(HttpCookie httpCookie, String responseBody, HttpResponse.HttpResponseBuilder response) {
+        if (httpCookie.containsJSessionId()) {
+            String sessionId = httpCookie.getJSessionId();
+            Session session = SessionManager.getInstance().findSession(sessionId);
+            if (session == null) {
+                buildOkResponse(responseBody, response);
+                return;
+            }
+            buildRedirectResponse("/index.html", response);
+        }
+
+        if (!httpCookie.containsJSessionId()) {
+            buildOkResponse(responseBody, response);
+        }
```

Comment:
이미 앞에서 분기처리가 되었으니, 마지막 if문은 제거해도 좋을 것 같아요 🙂
```suggestion
            buildRedirectResponse("/index.html", response);
            return;
        }

        buildOkResponse(responseBody, response);
```